python/pylibcudf/pylibcudf/libcudf/transform.pxd

```diff
@@ -40,13 +40,6 @@ cdef extern from "cudf/transform.hpp" namespace "cudf" nogil:
         device_memory_resource* mr
     ) except +libcudf_exception_handler
 
-    cdef unique_ptr[column] compute_column(
```

Comment:
Hmm I thought for sure I already removed this duplicate entry... anyway, thanks.